composer.json

```diff
@@ -38,10 +38,10 @@
     "require": {
         "php": ">=7.2.0",
         "doctrine/dbal": "^2.6.0",
-        "symfony/console": "~4.1",
-        "symfony/event-dispatcher": "~4.1",
-        "symfony/process": "~4.1",
-        "symfony/yaml": "~4.1"
+        "symfony/console": "~3.0|~4.1",
```

Comment:
3.0 is EOL, so we shouldn't support it.
We should only support 3.4 (LTS)

https://symfony.com/roadmap?version=3.4#checker